test/EnvironmentManagerTests/EnvironmentManagerUWPTests.h

```diff
@@ -8,40 +8,47 @@ namespace ProjectReunionEnvironmentManagerTests
 {
     class EnvironmentManagerUWPTests {
         BEGIN_TEST_CLASS(EnvironmentManagerUWPTests)
-            TEST_CLASS_PROPERTY(L"RunAs", L"{UAP,Restricted}")
+            TEST_CLASS_PROPERTY(L"RunAs", L"UAP")
             TEST_CLASS_PROPERTY(L"UAP:AppXManifest", L"AppxManifest.pkg.xml")
-            TEST_CLASS_PROPERTY(L"RunFixtureAs", L"Elevated")
+            TEST_CLASS_PROPERTY(L"RunFixtureAs", L"ElevatedUser")
             END_TEST_CLASS()
 
-            TEST_CLASS_SETUP(UWPWriteEVs)
+        TEST_CLASS_SETUP(UWPWriteEVs)
         {
-            WriteProcessEV();
             WriteUserEV();
             WriteMachineEV();
-
             return true;
         }
 
-        TEST_CLASS_CLEANUP(UWPRemoveEvs)
+        TEST_CLASS_CLEANUP(IWPRemoveEVs)
         {
-            RemoveProcessEV();
```

Comment:
EV? Ah! EnvironmentVariable.

Minor Suggestion: Rename *RemoveEV(s) to *Cleanup() would avoid the abbreviation